server/src/main/java/com/objectcomputing/checkins/services/skills/SkillServices.java

```diff
@@ -4,38 +4,43 @@
 import org.slf4j.LoggerFactory;
 
 import javax.inject.Inject;
-import java.util.Collections;
+import javax.inject.Singleton;
 import java.util.List;
 import java.util.UUID;
 import java.util.stream.Stream;
 
+@Singleton
 public class SkillServices {
 
     private static final Logger LOG = LoggerFactory.getLogger(SkillServices.class);
 
     @Inject
-    private SkillRepository skillsRepo;
+    private SkillRepository skillRepository;
+
+    public void setSkillRepository(SkillRepository skillsRepository) {
+        this.skillRepository = skillsRepository;
+    }
 
     protected Skill saveSkill(Skill skill) {
 
-        List<Skill> returnedList = findByValue(null, skill.getName(), null);
-        return returnedList.size() < 1 ? skillsRepo.save(skill) : null;
+        List<Skill> returnedList = findByValue(skill.getName(), null);
+        return returnedList.size() < 1 ? skillRepository.save(skill) : null;
 
     }
 
     protected Skill readSkill(UUID skillId) {
 
-        Skill returned = skillsRepo.findBySkillid(skillId);
+        Skill returned = skillRepository.findBySkillid(skillId);
 
         return returned;
 
     }
 
-    protected List<Skill> findByValue(UUID skillid, String name, Boolean pending) {
+    protected List<Skill> findByValue(String name, Boolean pending) {
+        LOG.info("findByValue");
```

Comment:
This log statement still needed, or should it bet set to debug?

Reply:
not really needed. will set to debug.